hasher-matcher-actioner/src/OpenMediaMatch/tests/test_api.py

```diff
@@ -221,6 +223,26 @@ def test_banks_add_hash_index(app: Flask, client: FlaskClient):
     assert post_response.json == {"matches": [2]}
 
 
+def test_lookup_add_hash_without_role(app: Flask, client: FlaskClient):
+    # role resets to True in the next test
+    client.application.config["ROLE_HASHER"] = False
+
+    # test GET
+    image_url = "https://github.com/facebook/ThreatExchange/blob/main/pdq/data/bridge-mods/aaa-orig.jpg?raw=true"
+    get_resp = client.get(f"/m/lookup?url={image_url}")
```

Comment:
blocking: I don't see any existing unittests in this file that check against this endpoint at all (the others are using "raw_lookup". Can you also test a lookup is accepted?

This file is getting big enough that it may make sense to break out unittests for the different APIs.